functions/Get-DbaEndpoint.ps1

```diff
@@ -66,6 +66,12 @@ function Get-DbaEndpoint {
                 Stop-Function -Message "Error occurred while establishing connection to $instance" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
             }
 
+            # the next block is the best we can do for docker machines that don't usually resolve in DNS
+            # if this code block is placed after Get-DbaEndpoint, everything fails. Not sure why, yet.
+            if ($server.NetName -and $server.HostPlatform -eq "Linux" -and $instance.ComputerName -notmatch '\.') {
+                Add-Member -InputObject $instance -NotePropertyName ComputerName -NotePropertyValue $server.NetName -Force
```

Comment:
What if you don't change $instance.ComputerName but change the if-statement down there with just another elseif ($server.NetName -and $server.HostPlatform -eq "Linux") { $dns = $server.NetName } ? That would be much clearer I think.

Reply:
It still get stuck. So weird. I'm guessing it has something to do with SMO and the way it adds replicas. 

![image](https://user-images.githubusercontent.com/8278033/93098450-25a89d00-f6a7-11ea-9aaf-a8127b4eed7a.png)